src/styles/blocks/card.scss

```diff
@@ -0,0 +1,81 @@
+@import "../utils/variables";
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 0;
+  transition-duration: 0.3s;
```

Comment:
You used transition duration here, but you should specify other transition-related declarations.